frameworks/helloworld/src/main/dist/examples/secrets.yml

```diff
@@ -0,0 +1,83 @@
+name: hello-world
+scheduler:
+  principal: {{SERVICE_PRINCIPAL}}
+  user: {{SERVICE_USER}}
+pods:
+  hello:
+    container:
+      image-name: ubuntu:14.04
+    count: {{HELLO_COUNT}}
+    placement: {{HELLO_PLACEMENT}}
+    secrets:
+      s1:
```

Comment:
It looks like these keys (s1, s2) are never used in the secret spec.  Should this be a list instead of a map?